src/coreclr/nativeaot/Runtime/thread.inl

```diff
@@ -2,6 +2,36 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 #ifndef DACCESS_COMPILE
+
+
+
+inline gc_alloc_context* ee_alloc_context::GetGCAllocContext()
+{
+    return (gc_alloc_context*)&m_rgbAllocContextBuffer;
+}
+
+inline uint8_t* ee_alloc_context::GetCombinedLimit()
+{
+    return combined_limit;
+}
+
+// It seems like there is a desire not to include a definition of gc_alloc_context in a more global place within
```

Comment:
This used to be the case in the early days of Redhawk. We do not care. Feel free to re-arrange things any way you want (bonus points for making things like field names same as in regular coreclr).

Reply:
Thanks for the info. I'm going to leave this PR as-is to reduce churn but I'll see if this is easy to improve upon in the followup PRs for this feature.